src/api/app/controllers/status/checks_controller.rb

```diff
@@ -26,6 +27,13 @@ def update
 
   private
 
+  def check_notify_params
```

Comment:
[nitpick] Are they not event_params?

Reply:
we call them notify_params everywhere else - they are parameters for the notification in my mind model :)

Reply:
Ok, fine then.